src/components/projects/ManuscriptPageContainer.tsx

```diff
@@ -48,6 +48,14 @@ export const ManuscriptMenusContainer = styled.div`
   display: flex;
   align-items: center;
   justify-content: space-between;
+  background-color: #fafafa;
+`
+export const ManuscriptMenusContainerInner = styled.div`
+  width: 100%;
+  max-width: 965px;
```

Comment:
Looks like 965 is derived from 960 + 5 so it would be nice to have that 960 as a variable or a part of the theme and use it here to just add 5px on top of it.

Reply:
I have added this dimension into theme
`...
grid {
  ...
  editorMaxWidth: 960,
...`

and have used it as:
`max-width: ${(props) => props.theme.grid.editorMaxWidth}px;`